File: backend/scripts/test-dns.py
import dns.resolver
try:
    answers = dns.resolver.resolve('_mongodb._tcp.cluster0.axzjobx.mongodb.net', 'SRV')
    for rdata in answers:
        print(f'Host: {rdata.target} Port: {rdata.port}')
except Exception as e:
    print(f'Error: {e}')

File: backend/scripts/test-dns.js
import dns from 'dns';

dns.resolveSrv('_mongodb._tcp.cluster0.axzjobx.mongodb.net', (err, addresses) => {
    if (err) {
        console.error('DNS Srv resolution error:', err);
    } else {
        console.log('Srv record addresses:', addresses);
    }
});
